.bazelci/presubmit.yml

```diff
@@ -237,21 +237,29 @@ tasks:
     name: compile_pip_requirements integration tests on Ubuntu
     working_directory: tests/compile_pip_requirements
     platform: ubuntu2004
+    run_targets:
+    - "bazel run //:requirements.update"
```

Comment:
is there any added test coverage from running this on every platform? I think of the lost rainforests when we waste CI resources 😉

Reply:
The experience has taught me that when we don't add tests for all platforms, it breaks. ;)

There are only 2 dependencies in there, and they should be cached, so the rainforests are safe (for now).